.github/workflows/haskell-ci.yml

```diff
@@ -87,6 +101,11 @@ on:
         type: boolean
         required: false
         default: true
+      hpc-flags:
+        description: Flags to pass to `hpc report`
```

Comment:
Perhaps this should just say "Flags to pass to `hpc`", given that we are also passing these flags to `hpc markup` in addition to `hpc report`.

Reply:
Good catch!